src/styles/blooks/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  max-width: calc(200px - 34px);
+  width: 100%;
+  font-size: 12px;
+  line-height: 18px;
+  padding: 16px;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)